hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -355,10 +355,6 @@
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
-        { bits: "3:0",
-          name: "RSVD",
-          desc: "Reserved bits."
-        },
```

Comment:
had this been allocated for anything specific?

Reply:
Ah I see, that was the abort thing. No worries, just wanted to understand what this was.